x-pack/plugins/screenshotting/server/browsers/chromium/paths.ts

```diff
@@ -67,23 +67,23 @@ export class ChromiumArchivePaths {
     {
       platform: 'linux',
       architecture: 'x64',
-      archiveFilename: 'chromium-70f5d88-locales-linux_x64.zip',
-      archiveChecksum: '759bda5e5d32533cb136a85e37c0d102',
+      archiveFilename: 'chromium-4440632-linux_x64.zip',
+      archiveChecksum: '2f88e9a17e156d27779e485dcce0946c',
       binaryChecksum: '82e80f9727a88ba3836ce230134bd126',
```

Comment:
We need to update the binaryChecksum as well, or else Kibana will re-download the archive at each startup in the dev environments